views/newChoral.pug

```diff
@@ -15,18 +15,22 @@ block content
     .form-group
       label(for='choralSampleRateInput') Sample Rate (seconds)
       .input-group
-        input#choralSampleRateInput.form-control(name='sampleRate' type='text', placeholder='5')
+        input#choralSampleRateInput.form-control(name='sampleRate' type='number', value='5')
         .input-group-addon.w3-blue seconds (>= 1)
-    .form-group
-      label(for='choralFuncInput') Function
-      textarea#choralFuncInput.form-control(name='func' type='text' placeholder='function(children, done){ done({ val: Math.random() * 50 }); }')
-      
-    if defaultFuncs.length > 0
+    #choralFuncInput
       .form-group
+        label(for='choralFuncInput') Function
+        textarea.form-control(name='func' type='text' )
+          | function(children, done){
+          |     done({ val: Math.random() * 50 });
+          | }
+
+    if defaultFuncs.length > 0
+      .form-group.inline-select
         label(for='choralDefaultFuncInput') Quick Function
-          select(name='defaultFunc' type='text').form-control#choralDefaultFuncInput
+          select(name='defaultFunc').form-control.add-child-input#choralDefaultFuncInput
```

Comment:
Is it possible to add the child on click, rather than having to press the "Add Child" button? That would get rid of the need for that warning output, and makes it less confusing for the user.